provider/terraform.rb

```diff
@@ -154,6 +154,9 @@ def generate_resource_tests(data)
       FileUtils.mkpath target_folder
       name = data[:object].name.underscore
       product_name = data[:product_name].underscore
+      ignore_read = data[:object].all_user_properties
```

Comment:
eh, since this is only used once I'd rather define it in the file where it gets used.